### ISSUE TITLE
Getting m3u8 error

### ISSUE THREAD

**Issue by jkaberg** [eithe/NRKTV.bundle#22]
So I just tried installing this channel, everything seems to work until I actually try to play the item where I get an "Cannot load m3u8: 404 not found"

Have you seen this yet? I get it both at the web player and while trying to cast to Chromecast

**Comment by eithe** [comment]
Hi, unfortunately I don't have the time to work on this plugin these days. Actually I'm also seeing the same, while for others the plugin works as expected. Not really sure about what causes it :(

**Comment by jkaberg** [comment]
@eithe Alright

I did some more research on this today and while I don't know why we get the error it seems the channel/plugin it self doesn't make use the official API(?)

So better would be to use the API, perhaps using [this](https://github.com/tamland/xbmc-addon-nrk/blob/master/nrktv.py) wrapper

**Comment by burnbay** [comment]
That's correct. It uses a mix of scraping and the old API. I don't have
time at the moment, but feel free to make changes :)

The plugin is not that relevant anymore after NRK shipped it's own app
where you can cast to Chromecast or Apple TV.

**Comment by jkaberg** [comment]
@burnbay yeah I know, I wanted to make use of the plugin to proxy NRK to friends outside of Norway :-)

Perhaps I'll take a stab at it, when I have more time™

**Comment by mikedebian** [comment]
I have a Plex server running on Debian and a raspberry 2 running Plexmediaplayer as the system. Just chiming in to say that I hope that you'll update it, so now you know theres one more :)
